src/glang/lib/std/aarch64_linux/syscalls.c3

```diff
@@ -68,3 +68,8 @@ function sys_munmap : (addr: iptr, length: isize) -> i64 = {
 function sys_exit : (code : int) -> void = {
     _syscall_1(code, /* exit */ 93);
 }
+
+function sys_assert_failed: () -> void = {
```

Comment:
Does this really need to be a target-specific function?

Reply:
I did this because I still want asserts on the gameboy but also I liked your changes from #170 better than just trapping everywhere. Moving this into the target dir seemed like a decent compromise.

I think in the long term we should be able to list target include directories in the actual target definition so all linux targets could do something like `include: posix` and we wouldn't need to duplicate this utility.

Reply:
Ok that's fair enough